### PR TITLE
Pull request for switchboard

### PR DESCRIPTION
**Pull request by fblackburn1**
Note: a small manual test has been added:  X-696 to verify wazo-calld auto-answer headers

POST-MERGE: remove testlink X-694

### CODE REVIEW

File: wazo_acceptance/helpers/call.py [hunk 1]
```diff
@@ -21,15 +23,18 @@ def start_recording(self, call_id):
...
+        caller_id_name = kwargs.pop('caller_id_name', _UNDEFINED)
         calls = self._calld_client.calls.list_calls(**kwargs)['items']
         for call in calls:
-            if user_uuid and user_uuid != call['user_uuid']:
-                continue
-            if caller_id_number and caller_id_number != call['caller_id_number']:
-                continue
-            if caller_id_name and caller_id_name != call['caller_id_name']:
-                continue
+            if user_uuid is not _UNDEFINED:
```
Comment (bloom1) [inline wazo_acceptance/helpers/call.py:31]: Why not use `None` and just check `if user_uuid is not None`?

Comment (bloom1) [reply wazo_acceptance/helpers/call.py:31]: Oh, alright, I had not seen the comment :smiley:

File: wazo_acceptance/steps/user.py [hunk 2]
```diff
@@ -54,18 +54,22 @@ def given_there_are_telephony_users_with_infos(context):
 
         confd_user = context.helpers.confd_user.create(body)
 
+        username = body.get('username') or context.helpers.utils.random_string(10)
+        password = body.get('password') or context.helpers.utils.random_string(10, sample=string.printable)
+        firstname = body['firstname']
+        lastname = body.get('lastname')
```
Comment (bloom1) [inline wazo_acceptance/steps/user.py:60]: Default value should be an empty string, this breaks other tests.

Comment (fblackburn1) [reply wazo_acceptance/steps/user.py:60]: fixed

**Review by bloom1 — APPROVED** [review]

**Comment by fblackburn1** [comment]
X-694 removed

**Pull request merged**